assets/css/style.css

```diff
@@ -0,0 +1,57 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+  }
+
+  @font-face {
+    font-family: poke;
+    src: url("./assets/fonts/pokemon-font.otf");
```

Comment:
@dvicj I think it should be: 
src: url("/assets/fonts/pokemon-font.otf")

Reply:
I think it should be fine either way with or without the dot but let me know if I should update 💯

Reply:
./ means current directory
/ means root directory

Reply:
Ahhh okay good point! I'll update it.

Reply:
I'm a little confused, is this your file or Devin's?